content/docs/user-guide/editor/viewport/group-selection.md

```diff
@@ -0,0 +1,76 @@
+---
+linkTitle: Group Selection
+title: Group Selection
+description: Select and manipulate groups of entities in the 3D Viewport in Open 3D Engine (O3DE).
+weight: 300
+---
+
+In the viewport, you can select multiple entities. This is called a group selection. You can use the following shortcuts for group selection.
+
+| Shortcut | Description |
+| --- | --- |
+| **Left-click** and **drag** in the viewport to create a box around entities. |  Selects multiple entities.  |
+|  Press and hold **Ctrl** and **left-click** and **drag** in the viewport.  | Deselect multiple entities. |
+|  Press and hold **Ctrl** and **left-click** a target entity.  |  Adds or removes an entity from the selection. If you already selected a group of entities, this moves the manipulator to the center of the selected group.  |
+
+When you select a group of entities, you can use the same controls as you would for a single entity. For a group selection, the manipulator acts like a temporary parent entity. This means that the feature works the same for a single entity or a group of entities. By default, a manipulator defaults to a common parent of the group selection. If you select entities that don't share a common parent, world space is used.
+
+## Selecting a group of entities
+
+1. In the viewport, select multiple entities.
+
+1. Press and hold **Ctrl** and **left-click** and **drag** the rotation manipulator.
```

Comment:
question: Should we mention at the start to me in rotation mode first?

Reply:
Good catch, this procedure doesn't match the intent of the section.  I changed the procedure to include adding entities to a group selection, and then removing entities from one.

Reply:
Perfect thanks!